.pre-commit-config.yaml

```
@@ -779,7 +779,7 @@ repos:
         files: ^\.pre-commit-config\.yaml$|^.*_installed_providers\.txt$
         pass_filenames: false
         require_serial: true
-      - id: update-spelling-wordlist-to-be-sorted
+      - id: update--wordlist-to-be-sorted
```

```suggestion
      - id: update-spelling-wordlist-to-be-sorted
```

I think this is still missed

---

Thank you. I have updated it. I wonder why I needed to do that?